ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -166,6 +182,10 @@ void shutdown() throws InterruptedException {
       final AtomicInteger numFailures = new AtomicInteger(0);
       List<Thread> shutdownThreads = new ArrayList<>();
       for (final Map.Entry<PartitionId, BlobStore> partitionAndStore : stores.entrySet()) {
+        if (stoppedReplicas.contains(partitionAndStore.getKey().toPathString())) {
+          // if the store is in stopped replica list, skip stop process and leave it in stopped state.
```

Comment:
I think the code makes this clear, comment is not required.

Reply:
removed